deployments/k8s/default/discovery-engine/configmap.yaml

```diff
@@ -91,7 +91,9 @@ data:
     logging:
       level: "INFO"
 
-    # kubectl -n kube-system port-forward service/hubble-relay --address 0.0.0.0 --address :: 4245:80
+    # kubectl -n kube-system port-forward service/hubble-relay --address 0.0.0.0
+    --address :: 4245:80
```

Comment:
Please verify this line

Reply:
Yes, I think it's mistakenly formatted. @vishnusomank Can you please remove it?